### PR TITLE
Id 135

### PR DESCRIPTION
**Pull request by JennaBarbara**

### CODE REVIEW

File: angular/src/app/shared/data-table/data-table.component.html [hunk 1]
```diff
@@ -13,3 +14,9 @@
                  [multiSortKey]="defaultTableConfig.multiSortKey"
                  [navigateToNextCell]="navigateToNextCell">
 </ag-grid-angular>
+<div style="align-self: flex-end;">
```
Comment (shiroyuki) [inline angular/src/app/shared/data-table/data-table.component.html:17]: Please see "semantic CSS" in https://docs.google.com/document/d/1fQz5u6oM56qZ2hinDvPklexG6qvnu-jcymdhU5NblVQ/edit

File: angular/src/app/shared/data-table/data-table.component.html [hunk 2]
```diff
@@ -13,3 +14,9 @@
...
+  <button (click)="onBtnExportDataAsCsv()" mat-stroked-button color="accent">
+    Download as CSV
+  </button>
```
Comment (shiroyuki) [inline angular/src/app/shared/data-table/data-table.component.html:20]: 1. According to the design, it is an icon button (`mat-icon-button`).
2. The name of the icon in question should be found in https://material.io/resources/icons/?style=baseline.

File: angular/src/app/shared/data-table/data-table.component.ts [hunk 3]
```diff
@@ -110,6 +110,17 @@ export class DataTableComponent implements OnDestroy {
...
+  onBtnExportDataAsCsv() {
+    const params = this.getParams();
+    this.gridApi.exportDataAsCsv(params);
+  }
+
+  getParams() {
+    return {
+      allColumns: true,
+    };
+  }
```
Comment (shiroyuki) [inline angular/src/app/shared/data-table/data-table.component.ts:122]: Can you merge this with `onBenExportDataAsCsv`?

**Review by shiroyuki — CHANGES_REQUESTED** [review]

File: angular/src/app/shared/data-table/data-table.component.html [hunk 4]
```diff
@@ -13,3 +14,9 @@
                  [multiSortKey]="defaultTableConfig.multiSortKey"
                  [navigateToNextCell]="navigateToNextCell">
 </ag-grid-angular>
+<div style="align-self: flex-end;">
+  <button (click)="onBtnExportDataAsCsv()" mat-stroked-button color="accent">
```
Comment (mpanik) [inline angular/src/app/shared/data-table/data-table.component.html:18]: should be just named `exportDataAsCsv()`

**Comment by mpanik** [comment]
Superseded by https://github.com/DNAstack/ddap-explore/pull/31

**Pull request closed without merging**